regression-test/suites/nereids_function_p0/scalar_function/L.groovy

```diff
@@ -91,4 +91,8 @@ suite("nereids_scalar_fn_L") {
 	qt_sql_ltrim_Varchar_notnull "select ltrim(kvchrs1) from fn_test_not_nullable order by kvchrs1"
 	qt_sql_ltrim_String "select ltrim(kstr) from fn_test order by kstr"
 	qt_sql_ltrim_String_notnull "select ltrim(kstr) from fn_test_not_nullable order by kstr"
+	qt_sql_ltrim_Varchar_Varchar "select ltrim(kvchrs1, 'constant') from fn_test order by kvchrs1"
```

Comment:
this case can not verify the function of ltrim.
It would be better if we replace "constant" to "var"
select ltrim(kvchrs1, 'var') from fn_test order by kvchrs1
then we get output "char11" instead of original value "varchar11"